pkg/webui/console/components/payload-formatters-form/index.js

```diff
@@ -187,6 +187,7 @@ class PayloadFormattersForm extends React.Component {
             component={Radio.Group}
             onChange={this.onTypeChange}
           >
+            {allowReset && <Radio label={sharedMessages.default} value={TYPES.DEFAULT} />}
```

Comment:
I think we should use another label, like `Use application payload formatter`, otherwise it's still ambiguous.
Then we can also remove the info notification and update the application level info notification to `Application level payload formatters are the default for all end devices. End device level payload formatters can be used to overwrite this default individually.`.

Reply:
Changed `Default` to `Use application payload formatter`. For the rest we can do that in a separate PR. I created an issue to improve the payload formatters pages in general, see https://github.com/TheThingsNetwork/lorawan-stack/issues/2463